gradle/build.gradle.kts

```diff
@@ -1,23 +1,19 @@
 val ktlint by configurations.creating
 
 repositories {
-	// artipie - an open source project that is used to store ktlint and diktat dependencies
     mavenCentral()
     jcenter()
-    maven {
-        url = uri("https://central.artipie.com/akuleshov7/diktat")
-    }
 }
 
 dependencies {
-    ktlint("com.pinterest:ktlint:0.37.1-fork") {
-	    // need to exclude standard ruleset to use only diktat rules
-	    exclude("com.pinterest.ktlint", "ktlint-ruleset-standard")
-	}
-	
-	// diktat ruleset 
-	ktlint("org.cqfn.diktat:diktat-rules:1.0.1") {
-		    exclude("org.slf4j", "slf4j-log4j12")
+    ktlint("com.pinterest:ktlint:0.37.1") {
+        // need to exclude standard ruleset to use only diktat rules
+        exclude("com.pinterest.ktlint", "ktlint-ruleset-standard")
+    }
+
+    // diktat ruleset
+    ktlint("org.cqfn.diktat:diktat-rules:0.1.0") {
+        exclude("org.slf4j", "slf4j-log4j12")
```

Comment:
It should be not needed to exclude slf4j anymore, it was needed because previously `diktat-rules` was jar with embedded dependencies, but now it is not. Same for maven.